celery_dyrygent/workflows/workflow.py

```diff
@@ -605,15 +619,20 @@ def freeze(self):
         self.logger.info("Workflow signature ready")
         return signature
 
-    def apply_async(self):
+    def apply_async(self, **kwargs):
```

Comment:
Perhaps it would be better to use
```
def apply_async(self, options=None)
```
So in the future it would be possible to pass some additional arguments.
What do you think?

Reply:
Thanks for the quick feedback man! 
So regarding both your comments, that's pretty much the only thing I wasn't sure about. I think we should support apply_async with the same signature as celery, which means using **kwargs. I don't think it should hurt us in the future while making changes, but I'm not 100% about that, so whatever you think :)

(Note that while initializing a Workflow object I do use options=dict(...), so your suggestion has the advantage of uniformity)

Reply:
I know celery uses **kwargs, but I don't think that's a good design decision.
Let's go with options=dict() :)

Reply:
no problem, changed :)

Reply:
Hey man, waiting for your approval :)

Reply:
Sure thing. 
I'd like to merge https://github.com/ovh/celery-dyrygent/pull/6 first.
And then release both changes under 0.7.0.
Besides I'm testing your changes right now :)

Reply:
Cool, they were indeed minimal.
I changed custom_payload to use the straight_serialzible default mechanism, I think it's neater.
No other changes

Reply:
Thx. The straight_serialzible default mechanism is pretty cool.